test-data/unit/check-parameter-specification.test

```diff
@@ -1296,3 +1296,26 @@ class C(Generic[P]):
 
 reveal_type(bar(C(fn=foo, x=1)))  # N: Revealed type is "__main__.C[[x: builtins.int]]"
 [builtins fixtures/paramspec.pyi]
+
+[case testParamSpecClassConstructor]
+from typing import ParamSpec, Callable
```

Comment:
```suggestion
[case testParamSpecClassConstructor]
# flags: --strict-optional
from typing import ParamSpec, Callable
```

This doesn't actually impact the crash, but I'd prefer to test with `--strict-optional` since that's the mypy default and it could conceivably make a difference (with `--no-strict-optional` NoneType is a lot like UninhabitedType).